stage1/enter_kvm/enter_kvm.go

```diff
@@ -223,17 +224,23 @@ func execSSH() error {
 
 	// this should not return in case of success
 	err = syscall.Exec(sshPath, args, os.Environ())
-	return fmt.Errorf("cannot exec to ssh: %v", err)
+	return errwrap.Wrap(errors.New("cannot exec to ssh"), err)
 }
 
 func main() {
 	flag.Parse()
+
+	if !debug {
+		log.setoutput(ioutil.discard)
```

Comment:
Does this even compile? I don't see any `log` object created in either local or global scope in this package nor any `log` package being imported.